src/main/java/karuna/karuna_backend/visitor/message/dto/VisitorMessageRequest.java

```diff
@@ -0,0 +1,11 @@
+package karuna.karuna_backend.visitor.message.dto;
+
+import org.springframework.data.domain.PageRequest;
+
+public record VisitorMessageRequest(int offset,
+                                    int limit,
+                                    int bodyLenLimit) {
+    public PageRequest toPageRequest() {
```

Comment:
No!! 
public record VisitorMessageRequest(PageRequest request,
                                    int bodyLenLimit)  like this!